ggml.c

```diff
@@ -4034,6 +4035,7 @@ struct ggml_tensor * ggml_new_tensor_impl(
         /*.grad         =*/ NULL,
         /*.src0         =*/ NULL,
         /*.src1         =*/ NULL,
+        /*.src2         =*/ NULL,
```

Comment:
Can you make it use `opt` instead of `src2`?

The problem with `src2` is that there is logic for constructing computation graphs that currently considers `src0`, `src1` and `opt`. So it would need to be updated if we introduce `src2`, but I want to do that at a later stage

Reply:
sure, I will change that